app/components/publications/publication/case/contact-person-edit-modal.hbs

```diff
@@ -0,0 +1,94 @@
+<Auk::Modal @size="medium"
+  data-test-add-contactpersonmodal>
+  <Auk::Modal::Header
+    @title={{t "add-contact"}}
+    @onClose={{@onCancel}} />
+  <Auk::Modal::Body>
+    <Utils::LoadableContent @isLoading={{this.isLoading}} @reserveSpace={{true}}>
+      <div class="auk-form-group-layout auk-form-group-layout--standard">
+        <div class="auk-form-group auk-form-group--grid">
+          <div class={{if this.validators.firstName.showError "auk-form-group--error"}}>
+            <Auk::Label for="firstNameInput" @required={{true}}>
+              {{t "first-name"}}
+            </Auk::Label>
+            <Auk::Input type="text" id="firstNameInput"
+              @block={{true}}
+              @value={{this.firstName}}
+              {{on "input" this.validators.firstName.enableError}} />
+          </div>
+          <div class={{if this.validators.lastName.showError "auk-form-group--error"}}>
+            <Auk::Label for="lastNameInput" @required={{true}}>
+              {{t "last-name"}}
+            </Auk::Label>
+            <Auk::Input type="text" id="lastNameInput"
+              @block={{true}}
+              @value={{this.lastName}}
+              {{on "input" this.validators.lastName.enableError}} />
+          </div>
+        </div>
+        <div class="auk-form-group">
+          <Auk::Label for="emailInput">{{t "email"}}</Auk::Label>
+          <Auk::Input type="text" id="emailInput"
+            @block={{true}}
+            @value={{this.email}} />
+        </div>
+        <div class="auk-form-group">
+          <Auk::Label>
+            {{t "organization"}}
+          </Auk::Label>
+          <div class="auk-o-flex">
+            <PowerSelect
+              @options={{this.organizations}}
+              @selected={{this.organization}}
+              @searchField="name"
+              @noMatchesMessage={{t "organizations-empty-result"}}
+              @placeholder={{t "search-organization"}}
+              @onchange={{this.setOrganization}}
+              @allowClear={{true}}
+              as |organization|>
+              {{ organization.name }}
+            </PowerSelect>
+            <Auk::Button
+              @layout="icon-only" @icon="add"
+              class="auk-u-ml-2"
+              {{on "click" this.openOrganizationAddModal}}>
+              {{t "add-organization"}}
+            </Auk::Button>
+          </div>
+        </div>
+      </div>
+    </Utils::LoadableContent>
+  </Auk::Modal::Body>
+  <Auk::Modal::Footer @custom={{true}}>
+    <Auk::Toolbar>
+      <Auk::Toolbar::Group @position="left">
+        <Auk::Toolbar::Item>
+          <Auk::Button
+            @skin="borderless"
```

Comment:
hier kunnen we geen template gebruiken door die `@disabled` (mogelijk moeten later die footer template eens uitbreiden ...)